### PR TITLE
Fix custom object renderer for buttons

### PR DESCRIPTION
**Pull request by D8H**
It's not a big extraction, but it's very safe and it will allow to see how buttons resizing works in the IDE with a nightly as I can't manage to build macOS version on the experimental branch.
The bonus is that the button won't appear broken in the IDE of previous GDevelop version when we launch it.

### CODE REVIEW

File: newIDE/app/src/ObjectsRendering/Renderers/RenderedCustomObjectInstance.js [hunk 1]
```diff
@@ -161,21 +176,25 @@ export default class RenderedCustomObjectInstance extends RenderedInstance {
...
+          const renderer = ObjectsRenderingService.createNewInstanceRenderer(
             project,
             layout,
             // $FlowFixMe Use real object instances.
             childInstance,
             childObjectConfiguration,
             this._pixiObject
           );
+          if (renderer instanceof RenderedTextInstance) {
+            renderer._pixiObject.style.align = 'center';
```
Comment (4ian) [inline newIDE/app/src/ObjectsRendering/Renderers/RenderedCustomObjectInstance.js:195]: Could this be done as a property of the text object?

File: newIDE/app/src/ObjectsRendering/Renderers/RenderedCustomObjectInstance.js [hunk 2]
```diff
@@ -229,32 +248,43 @@ export default class RenderedCustomObjectInstance extends RenderedInstance {
...
+      : this.getDefaultHeight();
+
     for (
       let index = 0;
       index < this.childrenRenderedInstances.length;
       index++
     ) {
       const renderedInstance = this.childrenRenderedInstances[index];
       const childInstance = this.childrenInstances[index];
-      childInstance.x = (defaultWidth - renderedInstance.getDefaultWidth()) / 2;
```
Comment (4ian) [inline newIDE/app/src/ObjectsRendering/Renderers/RenderedCustomObjectInstance.js:239]: For now we can do like this, but in the future we could probably add a setting to choose for each child object instances if they are stretched or centered when rendered in the scene editor.
Could help for some basic cases like buttons or a few other simple objects.

**Review by 4ian — APPROVED** [review]

File: newIDE/app/src/ObjectsRendering/Renderers/RenderedCustomObjectInstance.js (hunk 2 above)
Comment (D8H) [reply newIDE/app/src/ObjectsRendering/Renderers/RenderedCustomObjectInstance.js:239]: This should probably be the default layout when there is no `initialInstances` defined.

Comment (4ian) [reply newIDE/app/src/ObjectsRendering/Renderers/RenderedCustomObjectInstance.js:239]: Ok!

File: newIDE/app/src/ObjectsRendering/Renderers/RenderedCustomObjectInstance.js (hunk 1 above)
Comment (4ian) [reply newIDE/app/src/ObjectsRendering/Renderers/RenderedCustomObjectInstance.js:195]: Let's add a "TODO: this should be a property added to the text object. When this property is added, remove these special cases".

Or if it's really not too complex => add it to the text + TextObject editor.

**Pull request merged**